specification/monitor/resource-manager/Microsoft.Insights/preview/2022-08-01-preview/scheduledQueryRule_API.json

```diff
@@ -460,7 +460,7 @@
           "type": "boolean",
           "description": "The flag that indicates whether the alert should be automatically resolved or not. The default is true. Relevant only for rules of the kind LogAlert."
         },
-        "ruleResolveConfiguration": {
+        "resolveConfiguration": {
```

Comment:
>resolveConfiguration [](http://example.com/codeflow?start=9&length=20)

Is this not a breaking change to the customers using this API today ?

Here and other places of the same change.

Reply:
@ramoka178 
ruleResolveConfiguration was a typo. 
resolveConfiguration is the actual name of the field and always was. 
JSON payloads using the ruleResolveConfiguration field name never worked: the field was not deserialized and was ignored.
We noticed it now and we would like to fix this error.